coralogix-eventbridge-policy/template.yaml

```diff
@@ -6,51 +6,67 @@ Parameters:
     Description: The ARN corresponding to the Event Bus that will receive events via the PutEvents method.
     AllowedPattern: '^arn:aws[\w-]*:events:[a-z]{2}-[a-z]+-[\w-]+:[0-9]{12}:event-bus\/[\.\-_A-Za-z0-9]+$'
     MaxLength: 2048
-  CoralogixRegion:
+  CoralogixRegionAlias:
     Type: String
-    Description: The Coralogix region, possible options are [US1, US2, EU1, EU2, AP1, AP2, Custom]
+    Description: The Alias for the Coralogix region, possible options are [us1, us2, eu1, eu2, ap1, ap2, custom]
     AllowedValues:
-      - US1
-      - US2
-      - EU1
-      - EU2
-      - AP1
-      - AP2
-      - Custom
+      - us1
```

Comment:
This value is provided from the web-app in uppercase. Did you verify if it's correctly applied?

Reply:
Yeah, I have the changes in the webapp ready too :)

Reply:
https://github.com/coralogix/cx-web-workspace/pull/766

Reply:
I was also trying to lowercase inside cloudformation but that was hell